src/test/java/dk/alexandra/fresco/suite/spdz/TestSpdzComparison.java

```diff
@@ -135,4 +138,32 @@ public void test_compareEQ_Sequential() throws Exception {
 		runTest(new ComparisonTests.TestCompareEQ(),
 				EvaluationStrategy.SEQUENTIAL, StorageStrategy.IN_MEMORY);
 	}
+	
+	@Test
+	public void test_isSorted() throws Exception {
+		runTest(new SortingTests.TestIsSorted(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	
+	@Test
+	public void test_compareAndSwap() throws Exception {
+		runTest(new SortingTests.TestCompareAndSwap(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	public void test_Sort() throws Exception {
+		runTest(new SortingTests.TestSort(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	@Ignore
+	public void test_Big_Sort() throws Exception {
+		runTest(new SortingTests.TestBigSort(),
+				EvaluationStrategy.SEQUENTIAL,StorageStrategy.IN_MEMORY);
+	}
+	@Test
+	public void test_logic() throws Exception {
```

Comment:
I'm still not sure what the logic thing does :) - What applications does it have?
Anyways, the test should not be within the comparison class. It should have it's own.